### PR TITLE
feat: add home to plugin marketplace

### PR DESCRIPTION
**Pull request by rubenLindstrom**
## Hey, I just made a Pull Request!
This PR adds the recently added Home Page plugin to the plugin marketplace.

#### :heavy_check_mark: Checklist
- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
Dude you snatched #7000!

**Comment by freben** [comment]
Also haha that icon, how much did it cost to commission?

File: microsite/data/plugins/home.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Home
+author: '@Spotify'
```
Comment (timbonicus) [inline microsite/data/plugins/home.yaml:3]:
```suggestion
author: Spotify
```
All of our plugins shown (except GRM) just have `Spotify` here.

**Review by timbonicus — CHANGES_REQUESTED** [review]
We should probably get our graphics department to make a nice logo here 😅  I can reach out to them.

**Comment by rubenLindstrom** [comment]
> Dude you snatched #7000!

Haha wow, almost didn't notice!

File: microsite/data/plugins/home.yaml (hunk 1 above)
Comment (rubenLindstrom) [reply microsite/data/plugins/home.yaml:3]: > We should probably get our graphics department to make a nice logo here 😅 I can reach out to them.

Haha, uploaded a new one now!

**Review by freben — APPROVED** [review]
Better!

**Pull request merged**